app/controllers/api/v1/invoices/find_all_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::V1::Invoices::FindAllController < ApplicationController
+  respond_to :json, :xml
+
+  def index
+    respond_with Invoice.where(strong_params)
+  end
+
+  private
```

Comment:
Should we name these specific to the model they are referring to. So invoice_params?